engine/src/main/java/org/terasology/engine/config/RenderingDebugConfig.java

```diff
@@ -19,6 +19,8 @@
     public static final String HUD_HIDDEN = "hudHidden";
     public static final String RENDER_CHUNK_BOUNDING_BOXES = "renderChunkBoundingBoxes";
     public static final String RENDER_SKELETONS = "renderSkeletons";
+    public static final String RENDER_ENTITY_BOUNDING_BOXES = "renderEntityBoundingBoxes";
```

Comment:
I renamed this since we're not rendering the entity colliders, but the bounding boxes as described by some component. Those two don't necessarily align, unfortunately.